core-container/entry.sh

```diff
@@ -218,6 +222,41 @@ function run_placement_translator() {
 }
 
 
+function run_dex() {
+    echo "--< Starting dex in background>--"
+    dex serve /home/kubestellar/dex/dex-config-dev.yaml &
+}
+
+
+function run_kb_backend() {
+    echo "--< Starting kube-bind example-backend in background >--"
+    kubectl ws root:espw
+    kubectl apply -f /home/kubestellar/kube-bind/deploy/crd
+    kubectl apply -f /home/kubestellar/kube-bind/deploy/examples/crd-mangodb.yaml
+    example-backend --oidc-issuer-client-id=kube-bind --oidc-issuer-client-secret=ZXhhbXBsZS1hcHAtc2VjcmV0 --oidc-issuer-url=http://127.0.0.1:5556/dex --oidc-callback-url=http://127.0.0.1:8080/callback --pretty-name="BigCorp.com" --namespace-prefix="kube-bind-" --cookie-signing-key=bGMHz7SR9XcI9JdDB68VmjQErrjbrAR9JdVqjAOKHzE= --cookie-encryption-key=wadqi4u+w0bqnSrVFtM38Pz2ykYVIeeadhzT34XlC1Y= --consumer-scope="Cluster" &
+}
+
+
+function run_kb_konnector_for_wmw() {
```

Comment:
Don't we want to do this in the ensure-wmw ?

Reply:
Yes, moved to ensure-wmw.